theano/sandbox/gpuarray/basic_ops.py

```diff
@@ -599,50 +681,53 @@ def c_code(self, node, name, inputs, outputs, sub):
     def c_code_cache_version(self):
         return (3,)
 
-
 cuda_from_gpu = CudaFromGpu()
 
 
 class GpuAlloc(HideC, Alloc):
-    def __init__(self, memset_0=False):
+    __props__ = ('memset_0', 'context') + Alloc.__props__
+
+    context_type = gpu_context_type
+
+    def __init__(self, context, memset_0=False):
```

Comment:
Not exactly, but keyword parameters have to go after regular ones, so I don't really have a choice there.